static/js/components/SearchBox.jsx

```diff
@@ -1,9 +1,20 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-import { useDispatch } from 'react-redux';
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
 
-import * as Actions from '../ducks/sources';
+import {
+  Paper,
+  Button,
+  ButtonGroup,
+  Checkbox,
+  FormControlLabel,
+  TextField,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
 
+import * as Actions from "../ducks/sources";
+
+import styles from "./SearchBox.css";
```

Comment:
can we get rid of these?

Reply:
I think I can get rid of the styles, but I am not sure about the Actions, since we need them to handle fetching sources.

Reply:
I think he just means the styles